app/controllers/invoices_controller.rb

```diff
@@ -6,5 +6,6 @@ def index
 
   def show
     @merchant_invoice = Invoice.find(params[:id])
+    @status_format = @merchant_invoice.status_format
```

Comment:
Nice addition here. I like that this was moved from the view and pushed to controller/model to handle